packages/core/features/src/signTransaction.ts

```diff
@@ -33,47 +34,39 @@ export type BitcoinSignTransactionVersion = '1.0.0';
  * @group SignTransaction
  */
 export type BitcoinSignTransactionMethod = (
-    input: BitcoinSignTransactionInput
-) => Promise<BitcoinSignTransactionOutput>;
+    ...inputs: readonly BitcoinSignTransactionInput[]
+) => Promise<readonly BitcoinSignTransactionOutput[]>;
 
 /**
  * Input for the {@link BitcoinSignTransactionMethod}.
  *
  * @group SignTransaction
  */
 export interface BitcoinSignTransactionInput {
-    /**
-     * Partially Signed Bitcoin Transaction (PSBT), as raw bytes.
-     */
+    /** Partially Signed Bitcoin Transaction (PSBT), as raw bytes. */
     readonly psbt: Uint8Array;
-    /**
-     * Chain to use.
-     */
-    readonly chain: ArrayElement<Wallet['chains']>;
-    /**
-     * Transaction inputs to sign.
-     */
+
+    /** Transaction inputs to sign. */
     readonly inputsToSign: InputToSign[];
-    /**
-     * Whether the wallet should broadcast the signed transaction.
-     */
-    readonly broadcast?: boolean;
-}
 
-/** A helper type to infer an array element type. */
-type ArrayElement<A> = A extends readonly (infer T)[] ? T : never;
+    /** Chain to use. */
+    readonly chain?: IdentifierString;
+}
 
-/** Transaction input to be signed with the specified {@link "@wallet-standard/base".WalletAccount.address}.
+/**
+ * Transaction input to be signed with the specified {@link "@wallet-standard/base".WalletAccount | account}.
  *
  * @group SignTransaction
  * */
 export interface InputToSign {
-    /** Address to use. */
-    address: WalletAccount['address'];
+    /** Account to use. */
+    readonly account: WalletAccount;
+
     /** List of input indexes that should be signed by the address. */
```

Comment:
nit: should we match the property name?
```suggestion
    /** List of input indexes that should be signed by the account. */
```